uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/annual-entitlements.component.spec.js

```diff
@@ -118,6 +119,17 @@ define([
       });
     });
 
+    describe('when user wants to see a comment to an entitlement', function () {
```

Comment:
Yeah, good point. Unfortunately I have to use `.error`. The notification service API provides methods: `success`, `error`, `alert` and `info`. Each method shows the connected icon, which has its own colour, for example, `error` shows **red** icon. And that is exactly the icon as per mockups in the original PR (https://github.com/civicrm/civihr/pull/1952). 

So, answering your question: we test _if user wants to see a comment, then a notification with a red icon should spawn_